cdp-agentkit-core/typescript/src/action_providers/basename/schemas.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+
+/**
+ * Input schema for registering a Basename.
+ */
+export const RegisterBasenameInput = z
```

Comment:
```suggestion
export const RegisterBasenameSchema = z
```

let's use the `Schema` postfix to make it align with the Action interface naming convention